samples/bluetooth/central_iso/README.rst

```diff
@@ -0,0 +1,24 @@
+.. _bluetooth_central_iso:
+
+Bluetooth: Central ISO
+######################
+
+Overview
+********
+
+Application demonstrating very basic BLE Central role functionality by scanning for other BLE
+devices and establishing a connection to the first one with a strong enough signal.
+The application then attempts to setup a connected isochronous channel and starts sending data.
+
+
+Requirements
+************
+
+* BlueZ running on the host, or
+* A board with BLE support
```

Comment:
BLE as an acr

Reply:
Just copied this from another sample; seems to be what almost all samples use, but I'll update it for this one.